python-checks/src/main/java/org/sonar/python/checks/JwtVerificationCheck.java

```diff
@@ -79,26 +80,30 @@ public void initialize(Context context) {
 
   private static void verifyCallExpression(SubscriptionContext ctx) {
     CallExpression call = (CallExpression) ctx.syntaxNode();
+
     Symbol calleeSymbol = call.calleeSymbol();
     if (calleeSymbol == null || calleeSymbol.fullyQualifiedName() == null) {
       return;
     }
-    if (WHERE_VERIFY_KWARG_SHOULD_BE_TRUE_FQNS.contains(calleeSymbol.fullyQualifiedName())) {
+
+    String calleeFqn = calleeSymbol.fullyQualifiedName();
+    if (WHERE_VERIFY_KWARG_SHOULD_BE_TRUE_FQNS.contains(calleeFqn)) {
       RegularArgument verifyArg = TreeUtils.argumentByKeyword("verify", call.arguments());
       if (verifyArg != null && Expressions.isFalsy(verifyArg.expression())) {
         ctx.addIssue(verifyArg, MESSAGE);
```

Comment:
Now, there is the possibility of raising 2 issues on `jwt.decode` calls that fail to meet both conditions. Given we raise on the argument, we could keep this as-is, but I think we'd need a test to show this is intentional.

Otherwise, maybe we could return here after raising the first issue, so that we don't raise twice?

Reply:
return added